ansys/dpf/post/result_data.py

```diff
@@ -256,48 +297,56 @@ def plot_contour(
         [{'elshape': 1, 'time': 1}, {'elshape': 0, 'time': 1}]
```

Comment:
common remark: get_all_label_spaces could be a good generic method for collections